playground/README.md

```diff
@@ -0,0 +1,187 @@
+# Playground
```

Comment:
Prefer not to call this `playground` as it's confusingly like the lit.dev playground but has different functionality and is for different use cases.

I think it would make the most sense to just integrate this with the existing `examples` folder, if possible.

Reply:
I see the use cases for this and the Lit.dev playground as highly aligned, actually. The Lit.dev playground is optimized for small, targeted examples that demonstrate a particular feature, technique, issue, etc. It tries to reduce the friction around creating and running such examples to as close to zero as possible, and that's exactly what this PR is trying to do in the context of the monorepo. I wanted the name of the folder in the monorepo to convey this similarity of purpose. For that reason, I briefly considered similar-but-not-quite-the-same names (e.g. "sandbox"), and am still open to something like that, but I found the benefit of association that came with using exactly the same name to outweigh whatever confusion might arise from the fact that the implementation is different.

In contrast, the things we currently have in the monorepo's `examples` folder are (at least currently) for demonstrating and testing integrations of Lit into various external stacks. They tend to be more complex, with nested subfolders, and each one has its own "infrastructure" (dev server, config files, etc.), whereas my vision for the monorepo playground is (again, like the Lit.dev playground) for the individual examples not to be concerned with any of that stuff.

If we are especially concerned about adding folders to the top level of the monorepo, I could imagine nesting `playground` (or whatever we end up deciding call it) inside `examples`, but its feels more like a peer than a child to me.